pages/support-us.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+
+    <link
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+      rel="stylesheet"
+      integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC"
+      crossorigin="anonymous"
+    />
+
+    <link
+      rel="stylesheet"
+      href="/hyf-redesign-group-1/styles/support-style.css"
+    />
+  </head>
+  <body>
+    <!--navbar-->
+    <header class="header">
+      <nav class="navbar navbar-expand-md bg-light navbar-light">
+        <div class="container">
+          <a href="index.html"
+            ><img
+              class="image"
+              src="/hyf-redesign-group-1/assets/hack-your-future-belgium-logo.svg"
+              alt="hyf-image"
+          /></a>
+
+          <button
+            class="navbar-toggler"
+            type="button"
+            data-bs-toggle="collapse"
+            data-bs-target="#navmenu"
+          >
+            <span class="navbar-toggler-icon">Toggle navigation</span>
+          </button>
+
+          <div class="collapse navbar-collapse" id="navmenu">
+            <ul class="navbar-nav ms-auto">
+              <li class="nav-item">
+                <a href="#program.html" class="nav-link pe-4">THE PROGRAM </a>
+              </li>
+
+              <li class="nav-item">
+                <a href="#volunteer" class="nav-link pe-4">VOLUNTEER</a>
+              </li>
+
+              <li class="nav-item">
+                <a href="#digitalents" class="nav-link pe-4">DIGITALENTS</a>
+              </li>
+
+              <li class="nav-item">
+                <a href="#about " class="nav-link pe-4">ABOUT</a>
+              </li>
+
+              <li class="nav-item">
+                <a href="#faq" class="nav-link pe-4">FAQ</a>
+              </li>
+
+              <li class="nav-item">
+                <a href="#support us" class="nav-link pe-4">SUPPORT US</a>
```

Comment:
in https://validator.w3.org/ it reports the following message
Bad value #support us for attribute href on element [a](https://html.spec.whatwg.org/multipage/#the-a-element): Illegal character in fragment: space is not allowed.

Reply:
i forget to link it , i will fix that👍